test/las_header_test.py

```diff
@@ -34,3 +35,50 @@ def test_las_header():
     assert las_header.evlr_offset == las_header.evlr_offset
     assert las_header.evlr_count == las_header.evlr_count
     assert las_header.points_by_return_14 == las_header.points_by_return_14
+
+
+def test_pickling():
+
+    reader = copc.FileReader("../test/data/autzen-classified.copc.laz")
+    las_header = reader.GetLasHeader()
+
+    data = pickle.dumps(las_header, -1)
+    las_header2 = pickle.loads(data)
+
+    assert las_header.file_source_id == las_header2.file_source_id
```

Comment:
should we make a serialize_test.py or something?

Reply:
Good idea haha